packages/neutrino-middleware-dev-server/index.js

```diff
@@ -0,0 +1,71 @@
+const ramda = require('ramda');
+const hot = require('neutrino-middleware-hot');
+const opn = require('opn');
+
+module.exports = (neutrino, options = {}) => {
+  if (process.env.NODE_ENV === 'development') {
+    const config = neutrino.config;
+    const server = ramda.pathOr({}, ['options', 'server'], neutrino);
+    const protocol = process.env.HTTPS ? 'https' : 'http';
+    const publicHost = process.env.HOST;
+    const port = process.env.PORT || server.port || options.port || 5000;
+    const https = (protocol === 'https') || server.https || options.https;
+    let openInBrowser = false;
+    let serverPublic = false;
+    let host = 'localhost';
+
+    if (server.public !== undefined) {
+      serverPublic = Boolean(server.public);
+    } else if (options.public !== undefined) {
+      serverPublic = Boolean(options.public);
+    }
+
+    if (serverPublic) {
+      host = '0.0.0.0';
+    }
+
+    if (server.open !== undefined) {
+      openInBrowser = Boolean(server.open);
+    } else if (options.open !== undefined) {
+      openInBrowser = Boolean(options.open);
+    }
+
+    neutrino.use(hot);
```

Comment:
Yes. I have recently discovered the real purpose of Hot Module Replacement. It may be used also in production for flawless deployment. I will move it back to the preset where it was before.